google/resource_google_organization_policy.go

```diff
@@ -238,6 +273,21 @@ func expandBooleanOrganizationPolicy(configured []interface{}) *cloudresourceman
 	}
 }
 
+func expandRestoreOrganizationPolicy(configured []interface{}) (*cloudresourcemanager.RestoreDefault, error) {
+	if len(configured) == 0 {
+		return nil, nil
+	}
+
+	restoreDefaultMap := configured[0].(map[string]interface{})
+	default_value := restoreDefaultMap["default"].(bool)
+
+	if default_value {
+		return &cloudresourcemanager.RestoreDefault{}, nil
+	}
+
+	return nil, fmt.Errorf("Invalid value for restore_policy. Expecting default = true")
```

Comment:
Instead of this, what do you think about adding a ValidateFunc to the field? That way, if someone sets the default to false, it would get caught at plan-time instead of apply-time.

Reply:
Added ValidateFunc